node/task.ts

```diff
@@ -1767,6 +1767,7 @@ export function findMatch(defaultRoot: string, patterns: string[] | string, find
 
 export interface ProxyConfiguration {
     proxyUrl: string;
+    proxyFormattedUrl: string;
```

Comment:
Resolved per offline discussion - since we considering that this is intended to be read-only object (containing current proxy settings) - it should be good to keep in a separate field